docs/learning/tutorials/use-existing-imodel.md

```diff
@@ -1,28 +1,13 @@
-# Online viewer
+# iTwin Hub
 
 If you already have access to an iModel this will guide you through locating the name of the Project, iModel, and viewing the iModel in an iModel.js based online viewer.

```

Comment:
"iTwin.js" instead of "iModel.js".  Probably worth looking around for more occurrences.